docs/usecase/usage-example.md

```diff
@@ -0,0 +1,64 @@
+# Use cases
+
+This document will introduce you to the example of what Vald can do.
+Vald is the approximate nearest neighbor dense vector search engine, which use [NGT](https://github.com/yahoojapan/NGT) as the core engine of Vald, and Vald manage to integrate with kubernetes.
```

Comment:
please add the word 'distributed' it's a very important part of vald

Reply:
fixed